packages/connect-react/src/hooks/useLoading.ts

```diff
@@ -0,0 +1,15 @@
+import { useState } from 'react';
+
+const LOADING_ANIMATION_TIMEOUT = 100;
+
+const useLoading = (initialState?: boolean) => {
+  const [loading, setLoading] = useState(initialState ?? true);
```

Comment:
I meant something different here :)
Most of the time the loading will be quite fast (it might take less than 100ms) 
=> it's confusing if we show a loading spinner for such a short time 
=> we should not show the spinner for the first 100ms, only if loading takes longer, we show it